ibm/service/dnsservices/data_source_ibm_private_dns_custom_resolver_forwarding_rules.go

```diff
@@ -91,16 +121,17 @@ func dataSourceIbmDnsCrForwardingRulesRead(context context.Context, d *schema.Re
 		forwardRule[pdnsCRFRType] = *instance.Type
 		forwardRule[pdnsCRFRMatch] = *instance.Match
 		forwardRule[pdnsCRFRForwardTo] = instance.ForwardTo
+		forwardRule[pdnsCRFRViews] = flattenPDNSFRViews(instance.Views)
```

Comment:
Please re-name the `instance` variable as it is misleading. This will help improve readibility

Reply:
Any suggestion for the new name?

Reply:
`rule`, `fRule`, `fwdRule` are all potential options

Reply:
It looks like you changed `forwardRule` instead of `instance`.

Reply:
fixed